examples/ziti-end-to-end/README.md

```diff
@@ -0,0 +1,177 @@
+# Setting up an End-to-End Ziti network
+## Prerequisites:
+
+Go 1.19 installed and setup: [download](https://go.dev/doc/install). You're going to want to set you `$GOPATH` env variable which can be anywhere on your system. This is where modules, packages, and installed binaries will end up. Putting `$GOPATH/bin` in your path will help with running the executables later.
+
+Ziti Binaries and relevant configs.
+
+[OpenZiti](https://github.com/openziti/ziti/tree/main) contains downloads for the latest builds or you can build it yourself. There are many example configs [in the etc dir](https://github.com/openziti/ziti/tree/main/etc).
+
+I have set up helpful environment variable for how I chose to get setup. I cloned the Ziti repository for local builds and set `$ZITI_SOURCE` to that directory. I then went into that directory and ran 
+```
+go install ./...
+```
+
+This will output the binary to `$GOPATH/bin`. I then do the same thing with this repository to install the go-httpbin binaries
```

Comment:
I overlooked this line in my test drive and so I didn't have the go-httpbin binary later on.

Reply:
Do you think it'd be better to explicitly call out the install and copy/paste the same info?

Reply:
Maybe one step per heading instead of multiple prerequisites, or maybe an ordered list of prerequisites would draw attention to each requirement.